### ISSUE TITLE
missing detector wiring mode + resistantces

### ISSUE THREAD

**Issue by petrleocompel** [petrleocompel/hikaxpro_hacs#60]
@iulius74
```
Hello. I vave now v 2.0.3 and it works perfect.
In model.py after each update I have to add to wiring mode:
class DetectorWiringMode(Enum):
SEOL = "SEOL"
DEOL = "DEOL"
TEOL = "TEOL"
NO_EOL = "No EOL"
I have TEOL on all detectors, and other way I got errors.

Also,

after line 660: fault_resistence: Optional[float] = None
after line 728: fault_resistence = from_union([from_float, from_none], obj.get("faultResistence"))
after line 781: fault_resistence,
after line 864: if self.fault_resistence is not None:
result["faultResistence"] = from_union([to_float, from_none], self.fault_resistence)
This way I got all resistors value.
Maybe you cand directly integrate this in code in future versions.
Thank you very much.
```

**Comment by petrleocompel** [comment]
`fault_resistence` is not documented in ISAPI docs.. again

**Comment by petrleocompel** [comment]
v2.1.0-beta4 released

**Comment by petrleocompel** [comment]
fixed again in v2.1.0-beta5
